src/App.vue

```diff
@@ -1,16 +1,18 @@
 <template>
   <div id="app">
     <Navigation />
+    <Header></Header>
```

Comment:
Den Header generell einzublenden und nur per v-if auszublenden finde ich aus Perfomance-Sicht problematisch. Ich schlage vor, ihn wirklich nur in den Seiten einzublenden, wo es nötig ist.